keyboards/ps2avrGB/README.md

```diff
@@ -10,11 +10,13 @@ Note that this is a complete replacement for the firmware, so you won't be
 using Bootmapper Client to change any keyboard settings, since not all the
 USB report options are supported.
 
+This is an example based on the b.mini keyboard for making other keyboards
+compatible with QMK; fully supported boards have their own directory.
+
 ## Supported Boards
 
-Only the [B.mini X2](http://winkeyless.kr/product/b-mini-x2-pcb/) has been
-tested so far (since it's the only one I own). But other boards that use
-the ps2avrGB firmware should work as well.
+- [b.fake](https://github.com/qmk/qmk_firmware/tree/master/keyboards/bfake)
+- [b.mini](https://github.com/qmk/qmk_firmware/tree/master/keyboards/bmini)
```

Comment:
Might be worth adding a reference to Pearl in here since I just successfully flashed it.

Reply:
I didn't know it was already ported and merged :) I've made an edit to add the reference.